### PR TITLE
Repo List - Style updates 

### PR DESCRIPTION
**Pull request by selfcontained**
## Description

Fixes a couple small styling issues on the repo list page.

* Tone down the prebuilds disabled icon & text
* Fix table headings to all be semibold.
* Adjust input and button heights/rounding to match each-other (reviewed this w/ Cory and did this to help push towards in flight design changes)

| Header | Header |
|--------|--------|
| image | image | 

* I also updated the sidebar for the repo config detail pages to remove Gitpod YAML for now (we cut it from initial scope) and hide the prebuild warning icon if prebuilds are enabled.

<details>
<summary>Summary generated by Copilot</summary>

### <samp>🤖[[deprecated]](https://githubnext.com/copilot-for-prs-sunset) Generated by Copilot at 70eb648</samp>

Updated the styling and font weights of some dashboard components to improve consistency and readability. Changed the alert icon color in `RepoListItem.tsx`, and applied `font-semibold` to the button and header text in `SortableTable.tsx` and `Table.tsx`.

</details>

## How to test

* Navigate to the Repositories menu item from the org menu
* Import a repo
* Navigate back to the list
* Verify the prebuilds disabled looks like the updated version in screenshot above
* Verify all of the table headers are the same font weight (semibold).

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - brad-repo-ea93d142e6</li>
	<li><b>🔗 URL</b> - <a href="https://brad-repo-ea93d142e6.preview.gitpod-dev.com/workspaces" target="_blank">brad-repo-ea93d142e6.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - brad-repo-list-style-updates-gha.20965</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-brad-repo-ea93d142e6%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-core-dev" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment / Integration Tests</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-large-vm
- [x] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [x] /werft preemptible
      Saves cost. Untick this only if you're really sure you need a non-preemtible machine.
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`. If enabled, `with-preview` and `with-large-vm` will be enabled.
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

File: components/dashboard/src/repositories/list/RepoListItem.tsx [hunk 1]
```diff
@@ -46,12 +45,10 @@ export const RepositoryListItem: FC<Props> = ({ configuration }) => {
                     {prebuildsEnabled ? (
                         <CheckCircle2Icon size={20} className="text-green-500" />
                     ) : (
-                        <AlertTriangleIcon size={20} className="text-red-500" />
+                        <AlertTriangleIcon size={20} className="text-kumquat-base" />
```
Comment (filiptronicek) [inline components/dashboard/src/repositories/list/RepoListItem.tsx:48]: Love that we're making this less scary, no prebuild shaming!

**Review by filiptronicek — APPROVED** [review]
Changes look good. SHIP IT

File: components/dashboard/src/components/podkit/tables/Table.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ export const TableHeader = React.forwardRef<HTMLTableSectionElement, React.HTMLA
         return (
             <thead
                 ref={ref}
-                className="[&_th]:p-3 [&_th]:bg-gray-100 dark:[&_th]:bg-gray-800 [&_th:first-child]:rounded-tl-md [&_th:last-child]:rounded-tr-md text-semibold"
+                className="[&_th]:py-2 [&_th]:px-4 [&_th]:bg-gray-100 [&_th]:font-semibold dark:[&_th]:bg-gray-800 [&_th:first-child]:rounded-tl-md [&_th:last-child]:rounded-tr-md"
```
[the remark this answers is not in the document]
Comment (selfcontained) [reply components/dashboard/src/components/podkit/tables/Table.tsx:30]: It applies the padding to any descendant `<th>` of the `<thead>` component.

**Pull request merged**